src/components/BulkEditPane/BulkEditListResult/PreviewLayout/EmbeddedTable/EmbeddedTable.js

```diff
@@ -1,41 +1,27 @@
 import PropTypes from 'prop-types';
-import css from './EmbeddedTable.css';
+import { DynamicTable } from '@folio/plugin-query-builder';
 
 export const EmbeddedTable = ({ value, headTitles }) => {
-  const tableBodyRows = value?.split('\u001f|')
-    .map(row => row.split('\u001f;'));
+  // Split rows by unit separator and cells by record separator + map to required format for DynamicTable
+  // \u001f and \u001e are non-printable characters used as delimiters
+  const values = value?.split('\u001f|')
+    .map(row => {
+      return row.split('\u001f;')
```

Comment:
Your comment mentions `\u001f` and `\u001e` but both the previous and current implementations only use `\u001f`. Is that intentional or is it a bug?